### PR TITLE
Stefan/add simpleapp

### PR DESCRIPTION
**Pull request by ssaroiu**
This PR adds a batched client and simple tests. 

Other than adding the new batched client class, no modifications have been made to the autorest module.

The simple tests illustrate how to use the Java library to make three types of calls to an ES server: a synchronous, an asynchronous, and a batched call.

There has been moderate testing done on the batch client. We tested using up to 10 GetBuildsCurrent calls (these calls require no auth). We also tested using a mix of GetBuildsCurrent and Post User calls (PostUser *does* require auth).

### CODE REVIEW

File: test/src/main/java/com/microsoft/test/embeddedsocial/AsyncTest.java [hunk 1]
```diff
@@ -0,0 +1,71 @@
+package com.microsoft.test.embeddedsocial;
```
Comment (acrown-msft) [inline test/src/main/java/com/microsoft/test/embeddedsocial/AsyncTest.java]: Should we add a copyright to these files?

File: test/src/main/java/com/microsoft/test/embeddedsocial/BatchTest.java [hunk 2]
```diff
@@ -0,0 +1,40 @@
+package com.microsoft.test.embeddedsocial;
+
+import com.microsoft.embeddedsocial.autorest.BuildsOperations;
```
Comment (acrown-msft) [inline test/src/main/java/com/microsoft/test/embeddedsocial/BatchTest.java]: Unnecessary import?

Comment (ssaroiu) [reply test/src/main/java/com/microsoft/test/embeddedsocial/BatchTest.java]: import removed.

File: autorest/src/main/java/com/microsoft/embeddedsocial/autorest/EmbeddedSocialBatchedClientImpl.java [hunk 3]
```diff
@@ -0,0 +1,281 @@
+package com.microsoft.embeddedsocial.autorest;
```
[the remark this answers is not in the document]
Comment (ssaroiu) [reply autorest/src/main/java/com/microsoft/embeddedsocial/autorest/EmbeddedSocialBatchedClientImpl.java]: done.

File: test/src/main/java/com/microsoft/test/embeddedsocial/AsyncTest.java (hunk 1 above)
Comment (ssaroiu) [reply test/src/main/java/com/microsoft/test/embeddedsocial/AsyncTest.java]: done.

**Pull request merged**